docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,107 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1 with Flexible Launch Control(FLC)
+
+## Install Git and Clone the Open Enclave SDK repo
+
+Download and install Git for Windows from [here](https://git-scm.com/download/win)
+
+Clone the Open Enclave SDK
+
+```powershell
+git clone https://github.com/openenclave/openenclave.gits
```

Comment:
Should this be .git, not .gits?

Reply:
`brmclare@BrettSGXDev:~/test$ git clone https://github.com/openenclave/openenclave.gits
Cloning into 'openenclave.gits'...
Username for 'https://github.com': ^C
brmclare@BrettSGXDev:~/test$ git clone https://github.com/openenclave/openenclave.git
Cloning into 'openenclave'...
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.`

Reply:
Fixed.